maintainers/maintainer-list.nix

```diff
@@ -3043,6 +3043,12 @@
     githubId = 9959940;
     name = "Andreas Fehn";
   };
+  felixsinger = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add felixsinger``.

Reply:
It is a seperate commit, see https://github.com/NixOS/nixpkgs/pull/111632/commits/a29eeb386efa82e51bf7af37ac4446932fc68083. Github just displays it in a confusing way.